src/containers/ClusterDetail/index.jsx

```diff
@@ -34,10 +36,10 @@ export default class ClusterDetail extends Component {
   }
 
   render() {
-    const { appStore } = this.props.rootStore;
+    const { clusterStore } = this.props;
     const { selectedRowKeys } = this.state;
 
-    const data = appStore.clusterNodes.items && [...appStore.clusterNodes.items] || [];
+    const data = clusterStore.clusterNodes.items && [...clusterStore.clusterNodes.items] || [];
```

Comment:
redundent logic, `items` is just the same as `[...items]` when items is an array

Reply:
@libangrui deep copy ?

Reply:
items is `ObservableArray`. It's Object in mobx.
But rc-table need the standard Array.

Reply:
ok, got it

Reply:
roger that.